be/src/exprs/agg/bitmap_union_int.h

```diff
@@ -3,6 +3,7 @@
 #pragma once
 
 #include "column/object_column.h"
+#include "column/type_traits.h"
```

Comment:
why add the header?

Reply:
If we introduce this header file separately, then we will get a compile error. This is very unfriendly to some IDEs.
so I add this header